testing/marketplace/deployer/helm_tiller_onbuild/helm-dependency-build/schema.yaml

```diff
@@ -0,0 +1,34 @@
+properties:
```

Comment:
If you could, please change the test directories to mirror the directory of the base deployer, instead of the make target. The goal is to easily find the source that this is testing, similar to `java` and `javatests` convention.

```
testing/marketplace/deployer_helm_tiller_base/onbuild/helm-dependency-build
testing/marketplace/deployer_helm_tiller_base/onbuild/standard
```

Reply:
Agreed - done!

Ideally the make targets will mirror the file system - hopefully we can align those in the future.